openid-sharedsignals-framework-1_0.md

```diff
@@ -1229,6 +1229,13 @@ Errors are signaled with HTTP status codes as follows:
 {: title="Create Stream Errors" #tablecreatestream}
 
 
+##### Validating a Stream Creation Response
+
+* `aud`: It is RECOMMENDED that the Receiver validates the `aud` in the Create Stream Response.
```

Comment:
I think you need to rephrase the sentence to use the word "SHOULD" (i.e., The Receiver SHOULD validate the 'aud' value in the Create Stream Response.

Reply:
```suggestion
* `aud`: the Receiver SHOULD validate the `aud` in the Create Stream Response.
```